core/arch/arm/arm.asm

```diff
@@ -468,8 +468,7 @@ GLOBAL_LABEL(dr_try_start:)
 GLOBAL_LABEL(dr_setjmp:)
         mov      REG_R2, ARG1
         stm      REG_R2!, {REG_R4-REG_R11, sp, lr}
-        vst1.64  {d8-d11},[REG_R2]!
-        vst1.64  {d12-d15},[REG_R2]!
+        vstmia   REG_R2!, {d8-d15}
```

Comment:
To be consistent with the line right above it, with the OP_vstm enum, etc., it would just be "vstm"